### PR TITLE
Update via web for 2.10/2.11 windows clients

### PR DESCRIPTION
**Pull request by bakulf**
This is the change in the addon to force the update via web for windows if the range is [2.10.0, 2.11.0]

### CODE REVIEW

File: addons/message_update_v2.12/update.js [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+      if (a[i] != b[i]) {
+        return a[i] > b[i] ? -1 : 1;
+      }
+    }
+    return 0;
+  }
+
+  if (versionCompare([2, 11, 1], version) >= 0 ||
+      versionCompare([2, 10, 0], version) < -1) {
+    api.navigator.requestScreen(api.navigator.ScreenUpdateRecommended);
```
Comment (oskirby) [inline addons/message_update_v2.12/update.js:29]: In this case, the balrog code should be working just fine. Shouldn't we simply doing nothing and leave the update recommendation screen to Balrog?

**Review by oskirby — COMMENTED** [review]
I have mild trepidation that we are going to pop open a browser session directly to the 2.12 update page every time the client is opened. Is there a mechanism to dismiss this, users may find this to be somewhat jarring.

File: addons/message_update_v2.12/update.js (hunk 1 above)
Comment (bakulf) [reply addons/message_update_v2.12/update.js:29]: > I have mild trepidation that we are going to pop open a browser session directly to the 2.12 update page every time the client is opened. Is there a mechanism to dismiss this, users may find this to be somewhat jarring.

this script runs only when the user clicks the button. This is the CTA for the button, see the manifest.json.

Comment (bakulf) [reply addons/message_update_v2.12/update.js:29]: > In this case, the balrog code should be working just fine. Shouldn't we simply doing nothing and leave the update recommendation screen to Balrog?

Correct. This is what that screen does. That screen triggers balrog and the self-update.

Comment (oskirby) [reply addons/message_update_v2.12/update.js:29]: Excellent, that addresses my concerns.

**Review by oskirby — APPROVED** [review]

**Pull request merged**